simulate/simulateIntergenic.go

```diff
@@ -35,7 +35,8 @@ const bufferSize = 10_000_000
 // lambda specifies the rate parameter for an exponential distribution, from which simulated INDEL sizes will be sampled.
 // gcContent specifies the expected value of GC content for inserted sequences.
 // vcfOutFile specifies an optional return, which records all variants made during the simulated mutation process.
-func SimulateWithIndels(fastaFile string, branchLength float64, propIndel float64, lambda float64, gcContent float64, vcfOutFile string) []fasta.Fasta {
+// transitionBias specifies the expected value of the ratio of transitions to transversions in the output sequence.
+func SimulateWithIndels(fastaFile string, branchLength float64, propIndel float64, lambda float64, gcContent float64, vcfOutFile string, transitionBias float64) []fasta.Fasta {
```

Comment:
Convention is to have input-associated variables before output-associated variables, so it might be more intuitive to have transitionBias come before vcfOutFile